packages/rest/src/rest.component.ts

```diff
@@ -89,6 +89,10 @@ export class RestComponent implements Component {
   ) {
     app.bind(RestBindings.SEQUENCE).toClass(DefaultSequence);
     const apiSpec = createEmptyApiSpec();
+    // Merge the OpenAPI spec from the config into the empty one
+    if (config && config.openApiSpec && config.openApiSpec.initialObject) {
```

Comment:
FYI: can now be: `config?.openApiSpec?.initialObject`
See [TS docs](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining)